Packet++/header/IPv4Layer.h

```diff
@@ -250,7 +250,7 @@ namespace pcpp
 
 				res.push_back(IPv4Address(curValue));
 
-				valueOffset += (uint8_t)(4);
+				valueOffset += 4;
```

Comment:
I think removing the casting might cause a compilation warning on some platfroms